recipes/r-readtext/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set version = '0.80' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-readtext
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/readtext_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/readtext/readtext_{{ version }}.tar.gz
+  sha256: be0016339eb081a60e26cbe9dd779695378edff8a2ce7b49809bbb94ae950a61
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [r_base == "4.0"]
```

Comment:
What is the reason for that skip? Not available dependencies?

Reply:
Dependency version inconsistencies/incompatibilities for r-base 4.0.

Reply:
The source of that could be the missing `r-antiword` package; possibly gets pulled in from somewhere else for 3.6. Added in https://github.com/conda-forge/staged-recipes/pull/13488 , please try the 4.0 build again once that is merged.